api/src/main/java/com/basistech/rosette/api/HttpRosetteAPI.java

```diff
@@ -152,7 +153,7 @@
         }
 
         mapper = ApiModelMixinModule.setupObjectMapper(new ObjectMapper());
-
+        mapper.configure(MapperFeature.CAN_OVERRIDE_ACCESS_MODIFIERS, false);
```

Comment:
HttpRosetteAPI.java has two constructors, one with two arguments (for testing) and the normal one with six arguments.  If I put this configuration line in the two-argument constructor, it causes a test failure.  I'm not sure why.  Moving the statement to ApiModelMixinModule would have the same effect.